packages/care-package/abstract-plugin/src/ProviderOrg.ts

```diff
@@ -1,31 +1,47 @@
-import {CloudDirectorConfig, OrgVdcNetworkApi, VdcApi} from '@vcd/node-client';
```

Comment:
Do we really want this common provider org code to live under abstract-plugin?

Reply:
Future plugins might use it as well, so a common place is expected.